apps/lead/filter_set.py

```diff
@@ -222,6 +234,15 @@ def authoring_organization_types_filter(self, qs, name, value):
             return qs.filter(organization_types__in=[ot.id for ot in value]).distinct()
         return qs
 
+    def filtered_exists_filter(self, qs, name, value):
+        if value == self.EXCLUDE_EMPTY_FILTERED_ENTRIES:
+            try:
+                # annotated in LeadViewSet
+                return qs.filter(filtered_entries_count__gte=1)
+            except FieldError:
+                return qs
```

Comment:
We have to make sure this is always provided. Let's remove try/except and make sure through test case.